### ISSUE TITLE
fields = '_all_' for django 1.7

### ISSUE THREAD

**Issue by ikresoft** [deschler/django-modeltranslation#292]
There is error when put in modelform fields = '__all__' to support django 1.7 I get error Unknown field(s) (a, l, _) specified for model.

**Comment by zlorf** [comment]
This constant require two underscores: ``'__all__'``

**Comment by ikresoft** [comment]
Yes I know, I didnt write it right here sorry, but in code is __all__, and error is the same

**Comment by zlorf** [comment]
I tried and cannot reproduce. Could you please provide some code extract? I assume that modelform is for translatable model, right?
